jinja2/includes/translate_locales.html

```diff
@@ -0,0 +1,62 @@
+This template is automatically generated by scripts/translate_locales.py to make the languages name localizable. Do not edit it manually
```

Comment:
I don't see a `scripts/translate_locales.py` file in this PR, is that the right file?